examples/xonotic/Makefile

```diff
@@ -84,13 +84,13 @@ push-windows-image-%:
 	$(MAKE) WINDOWS_DOCKER_PUSH_ARGS=--push build-windows-image-$*
 
 build-windows-image-%: ensure-windows-buildx
-	docker buildx build --platform windows/amd64 --builder $(BUILDX_WINDOWS_BUILDER) -f $(project_path)Dockerfile.windows \
-	--tag=$(server_tag)-windows_amd64-$* --build-arg WINDOWS_VERSION=$* . $(WINDOWS_DOCKER_PUSH_ARGS)
+		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform windows/amd64 --builder $(BUILDX_WINDOWS_BUILDER) -f $(project_path)Dockerfile.windows \
```

Comment:
Is `DOCKER_CLI_EXPERIMENTAL=enabled` needed? `docker buildx build` is not an experimental feature.

nit: why extra indentation is added?

Reply:
This PR addresses [this issue](https://github.com/googleforgames/agones/issues/3530)

[The build](https://gist.github.com/Kalaiselvi84/e82da307f81102e5449cb4c2c61cd511) for Xonotic was successful without enabling `DOCKER_CLI_EXPERIMENTAL`, but the build failed without this for the simple-game-server🤔

Reply:
There is some weirdness, since Cloud Build server is 20.x and we're using the Docker client 24.x -- so sometimes some things don't work as you expect them to with a full 24.x installation.

Reply:
Are we good with the updates without including the `DOCKER_CLI_EXPERIMENTAL=enabled` and `--use` flag? This is the expected update, correct?

Reply:
As long as it works, I don't mind 😃

Reply:
Cool, shall we proceed with merging this PR?🙂